documentation.css

```diff
@@ -36,17 +39,17 @@
 .header-nav-link{
     font-size: 2rem;
     margin-right: 2rem;
+    transition: 0.3s;
 }
 
 .header-nav-link:hover{
     color: var(--SECONDARY-COLOR);
-    transition: 0.3s;
 }
 /* ------- header end -------  */
 
 main{
     display: grid;
-    grid-template-columns: 22% 1fr 5%;
+    grid-template-columns: 25% 1fr 5%;
     grid-template-areas: 
     "side-nav documentation ...."
```

Comment:
I like what you have done here to get space on the right of the viewport. Any reason for not going with padding?

Reply:
I am not sure either. it practically achieves the purpose. I think it won't be an issue when making the site responsive either. Maybe ask someone in the server if they can opine.